.typo-ci.yml

```diff
@@ -0,0 +1,141 @@
+# This is a sample .typo-ci.yml file, it's used to configure how Typo CI will behave.
+# Add it to the root of your project and push it to github.
+---
+
+# What language dictionaries should it use? Currently Typo CI supports:
+# de
+# en
+# en_GB
+# es
+# fr
+# it
+# pt
+# pt_BR
+dictionaries:
+    - en
+    - en_GB
```

Comment:
are we sure it should be both en and en_GB? should probably just be one of the two across the docs

Reply:
that's fair I'll switch it on the test pr and see what happens

Reply:
I guess we should be consistent with our blog? So US english. Quite a few british spelling changes then need to happen, which is fine if everyone realizes that